doc/faq.rst

```diff
@@ -103,8 +103,8 @@ Practical
   The best ever designed numerical library - NumPy_.
 
 2. **Which Python console do you recommend?**
-  ``IPython 
-<http://www.ipython.org/>`_.`
+ `IPython`_
+.. _IPython: http://www.ipython.org/
```

Comment:
The link should be move to [links_names.inc](https://github.com/nipy/dipy/blob/master/doc/links_names.inc#L82)

Reply:
@skoudoro i added the link to links_names.inc but i am not sure if i have the permissions to squash and merge. Please correct me if i am wrong because i am a newbie and i dont know much about github